docs/src/extend/shareable-configs.md

```diff
@@ -12,6 +12,10 @@ To share your ESLint configuration, create a **shareable config**. You can publi
 
 This page explains how to create and publish a shareable config.
 
+::: tip
+This page explains how to create a shareable config using the flat config format. For the deprecated eslintrc format, [see the deprecated documentation](shareable-configs-deprecated).
+:::
+
 ## Creating a Shareable Config
 
 Shareable configs are simply npm packages that export a configuration object. To start, [create a Node.js module](https://docs.npmjs.com/getting-started/creating-node-modules) like you normally would.
```

Comment:
Re `The module name must take one of the following forms...`: the module name can be any valid npm package name, but the following forms are still recommended?

Reply:
Good catch. I'll update that.